docs/runbooks/NoReadyVirtController.md

```diff
@@ -1,52 +1,64 @@
+<!-- Edited by Jiří Herrmann, 9 Nov 2022 -->
+
 # NoReadyVirtController
 
 ## Meaning
 
-From a high-level perspective the virt-controller is responsible for all the cluster wide virtualization functionality.
-
-This controller is responsible for monitoring the VMI (CRDs) and managing the associated pods. The controller will make sure to create and manage the life-cycle of the pods associated to the VMI objects.
+This alert fires when no available `virt-controller` devices have been detected for 5 minutes.
 
-A VMI object will always be associated to a pod during it's life-time, however, due to i.e. migration of a VMI the pod instance might change over time.
+A `virt-controller` handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the lifecycle of the pods.
```

Comment:
In this context, I would use the generic form of virt-controller: "The virt controller handles....